### PR TITLE
Update pr_BR.json

### PR DESCRIPTION
**Pull request by mbitencourt**
Fix some terms and add new lines to support 7.5 version.

### CODE REVIEW

File: locales/pt_BR.json [hunk 1]
```diff
@@ -1,308 +1,316 @@
...
+    "OWNER_IS_YOU" : "Você é o Proprietário",
+    "NEW_TASK_ASSIGNEE" : "Responsável",
+    "VALUE" : "Valor",
+    "FILTER_DESCRIPTION_PLACEHOLDER" : "Mostrar as tarefas atribuídas a mim",
+    "TASK_VARIABLE" : "Variável de Tarefa",
+    "GROUP_ADD_ERROR" : "Grupo não pode ser adicionado",
+    "FILTER_SAVE_ERROR" : "Alguma erro ocorreu ao guardar o filtro",
+    "COMMENT_CREATE" : "Adicionar Comentário",
+    "BEFORE" : "antes",
+    "PROCESS_EXTERNAL_FORM_NOTE" : "Este processo contém uma forma externa para iniciar uma nova instância do processo. A fim de abrir a forma externa clique sobre o link abaixo. \"Abrir formuário externo\"",
```
Comment (zeropaper) [inline locales/pt_BR.json]: I think this should be "formulário" instead of "formuário".

File: locales/pt_BR.json [hunk 2]
```diff
@@ -1,308 +1,316 @@
...
+    "PROCESS_DEFINITION" : "Definição do Processo",
+    "RESTRICTION_NOTICE" : "Os atuais parâmetros de busca não conseguiram obter quaisquer tarefas. Tente redefinir os parâmetros da pesquisa.",
+    "SHORTCUT" : "Atalho",
+    "FILTER_SAVE_SUCCESS" : "O filtro foi salvo",
+    "ASSIGNED_OK" : "A tarefa foi atribuída.",
+    "DESERIALIZATION_ERROR" : "Erro de deserialização",
+    "INVALID_DATE" : "Data inválida: a data deve ter o padrão 'yyyy-MM-ddTHH:mm:ss'",
+    "AddGroupLink" : "Adiconar Link do Grupo",
+    "EXECUTION_VARIABLE" : "Variável de Execução",
+    "ADD_PERMISSION" : "Adicionar pemissão.",
```
Comment (zeropaper) [inline locales/pt_BR.json]: I think it should be "permissão" instead of "pemissão"

File: locales/pt_BR.json [hunk 3]
```diff
@@ -1,308 +1,316 @@
...
+    "FILTER_CRITERIA_INCLUDE_ASSIGNED_TASKS_HINT" : "Também incluem tarefas que são atribuídas a usuários em possíveis consultas<br\/>O padrão é para incluir apenas tarefas que são atribuídas a qualquer usuário, se você consultair pelo usuário candidato ou grupo(s).",
+    "FILTER_CRITERIA_VALUE" : "Valor",
+    "Suspend" : "Suspender",
+    "TASK_VARIABLES" : "Variáveis",
+    "delegation" : "Delegação",
+    "DELETE_SEARCH" : "Remover pesquisa",
+    "NAME" : "Nome",
+    "SetOwner" : "Definir Dono",
+    "FILTER_FORM_PERMISSIONS_DUPLICATE_USER" : "Atribuido permissão de leitura ao usuário",
+    "TASK_EXTERNAL_FORM_NOTE" : "Esta tarefa contém um formulário externo para completá-la. A fim de abrir a forma externa clique sobre o link abaixo. \"Abrir formuário externo\"",
```
Comment (zeropaper) [inline locales/pt_BR.json]: Typo again? :)
"formulário" instead of "formuário"

File: locales/pt_BR.json [hunk 4]
```diff
@@ -1,308 +1,316 @@
...
+    "SET" : "Set",
+    "DATE" : "Data",
+    "FILTER_FORM_PERMISSION_IDENTIFIER" : "Identificador",
+    "CREATION_DATE" : "Criado",
+    "SUCCESSFUL" : "Sucesso",
+    "FORM" : "Formulário",
+    "MY_PROFILE" : "Meu Perfil",
+    "FILTERS_LOADING_FAILURE" : "Carregamento dos filtros terminou com falhas. Tente atualizar a página para tentar novamente.",
+    "APP_VENDOR" : "Camunda",
+    "INIT_EXTERNAL_FORM_FAILED" : "Não é possível inicializar a URL do formuário externo: ou ID de tarefa ou chave de definição do processo devem ser fornecidos. Tente atualizar a página para tentar novamente.",
```
Comment (zeropaper) [inline locales/pt_BR.json]: Typo again? :)
"formulário" instead of "formuário"

**Comment by zeropaper** [comment]
Hello!

First of all: thank you very much for the PR!

I did noticed a few typos (it looks like some of them were already present in the previous version of the file).

Could you have a look at the commit comments and fix them (if necessary)?
That would be great :+1: :)

Thanks again and greetings from Camunda HQ,
Vale

File: locales/pt_BR.json (hunk 3 above)
Comment (mbitencourt) [inline locales/pt_BR.json]: Yes, "formulário" is correct term.
Sorry.

File: locales/pt_BR.json (hunk 2 above)
Comment (mbitencourt) [inline locales/pt_BR.json]: Yes, "permissão" is correct term.

**Comment by zeropaper** [comment]
Hi mbitencourt,

thanks for the clarifications!
can you make the changes in your pull request please?
Then I'll be very happy to accept it and merge.

Cheers,
Vale

**Comment by zeropaper** [comment]
I made the last corrections and merged your pull request

**Pull request closed without merging**